casper/src/main/scala/coop/rchain/casper/api/GraphGenerator.scala

```diff
@@ -168,7 +156,7 @@ object GraphzGenerator {
       validators: Map[String, ValidatorsBlocks]
   ): G[Unit] =
     validators.values.toList
-      .flatMap(_.values.toList)
+      .flatMap(_.values.toList.flatten)
```

Comment:
obviously an `extraFlatMap` operator is needed :))

Reply:
hehe :)